frontend/components/ClubPage/Events.js

```diff
@@ -0,0 +1,53 @@
+import { Icon, Text, StrongText, Card } from '../common'
+import { useState, useEffect } from 'react'
+
+const Events = props => {
+  const data = props.data
+
+  const iconStyles = {
+    float: "left",
+    marginRight: "7px"
+  }
+
+  const bigStyles = {
+    fontSize: "13px",
+    fontWeight: "bold"
+  }
+
+  const smallStyles = {
+    fontSize: "13px"
+  }
+
+  if (!data || !data.length) {
+    return <></>
+  }
+
+  return (
+    <Card bordered style={{ marginBottom: '1rem' }}>
+      <div style={{marginLeft: '7px'}}>
```

Comment:
What does this div do?

Reply:
I actually cannot remember why I added this so I've taken it out. Thanks for pointing it out.